swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -359,6 +362,7 @@ paths:
         - $ref: "#/components/parameters/Signature"
         - $ref: "#/components/parameters/TPP-Signature-Certificate"
         - $ref: "#/components/parameters/TPP-Redirect-Preferred"
+        - $ref: "#/components/parameters/TPP-Decoupled-Preferred"
```

Comment:
Relevant Change: TPP-Decoupled-Preferred is new
Adoption approach: Follow the change